src/gtil/predict.cc

```diff
@@ -161,21 +176,21 @@ inline std::size_t PredictImpl(const treelite::ModelImpl<ThresholdType, LeafOutp
           sum[i] += leaf_vector[i];
         }
       };
-      return PredictImplInner(model, input, output, pred_transform, output_logic);
+      return PredictImplInner(model, input, output, nthread, pred_transform, output_logic);
     } else {
       // multi-class classification with gradient boosted trees
       auto output_logic = [task_param](
           const TreeType& tree, int tree_id, int node_id, float* sum) {
         sum[tree_id % task_param.num_class] += tree.LeafValue(node_id);
       };
-      return PredictImplInner(model, input, output, pred_transform, output_logic);
+      return PredictImplInner(model, input, output, nthread, pred_transform, output_logic);
     }
   } else {
     auto output_logic = [task_param](
```

Comment:
Where is param being used?

Reply:
My bad. Let me remove it from the lambda capture.